main.tf

```diff
@@ -361,6 +361,12 @@ module "influxdb" {
   target_repository = "${var.target_repository}/influxdb"
 }
 
+module "istio-proxy" {
+  source            = "./images/istio-proxy"
+  target_repository = "${var.target_repository}/istio-proxy"
+}
```

Comment:
Can we create a single Istio directory and have this be a proxy component within it?

I assume we'll have multiple containers at some point.

Reply:
e.g. I could imagine us adding an `istioctl` image at least, but `istiod` seems like another logical one.

Reply:
Done